### PR TITLE
PR: Consolidate extract to single path

### LINKED ISSUE #160: ER: Use performance data for active standby database
### Enhancement Request Details

As a end user I want to get performance data from active standby databases that have been configured to record their data to the DBA_HIST_* tables in the primary database.

### Version

_No response_

### What oracle database version are you seeing the problem on?

12c and above

### Which sql script version ?

_No response_

### Relevant log output

_No response_

### LINKED ISSUE #172: ER: Need workaround for DBA_HIST_SYSMETRIC_SUMMARY.standard_deviation issue
### Enhancement Request Details

As an end user I want to handle cases where DBA_HIST_SYSMETRIC_SUMMARY.standard_deviation returns NULL.  There have been cases where DBA_HIST_SYSMETRIC_SUMMARY.standard_deviation displays as NULL. yet a numeric comparison has it between (0.9 repeating) and 1.  In these cases, assign 0.  Observed in 11.2.0.4 and 19.3.

### Version

1.0.2 (Default)

### What oracle database version are you seeing the problem on?

All

### Which sql script version ?

2.0.4

### Relevant log output

_No response_

### LINKED ISSUE #175: ER: Consolidate SQL Collection Scripts
### Enhancement Request Details

Combine the 11G & 12G scripts into a single set.

### Version

_No response_

### What oracle database version are you seeing the problem on?

_No response_

### Which sql script version ?

_No response_

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by wpuziewicz**
Take the two separate scripts for 11 and under vs 12 and up & makes them one set of scripts
Closes #172 
Partially resolves #160
Closes #175

### CODE REVIEW

**Comment by wpuziewicz** [comment]
Resolves #178

**Comment by wpuziewicz** [comment]
Closes #179

**Review by cofin — DISMISSED** [review]
Changes look good!

**Comment by cofin** [comment]
Before we merge this into `main`, @smancinijr and @smpawar , can you guys take a look at these changes?  I've done an initial review and these look good to me.

**Review by shane-borden — COMMENTED** [review]
In script "db_assessment/dbSQLCollector/collectData-Step1.sh", there is commented code that should likely be removed.

File: db_assessment/dbSQLCollector/collectData-Step1.sh [hunk 1]
```diff
@@ -27,13 +27,8 @@ dbVersion=$(( mainversion[0] + 0 ))
 
 full_path="$(dirname $(realpath $0))"
 SQL_SCRIPT=${full_path}
-if [ ${dbVersion} -ge 12 ]; then
+#if [ ${dbVersion} -ge 12 ]; then
```
[the remark this answers is not in the document]
Comment (wpuziewicz) [reply db_assessment/dbSQLCollector/collectData-Step1.sh:30]: Yes, will do.

**Comment by shane-borden** [comment]
Tested against version 19.15 and 12.2.0.1.  No issues on either test.

**Review by shane-borden — APPROVED** [review]
Reviewed and tested against 12.2.0.1 and 19.15 Oracle databases.

**Review by shane-borden — COMMENTED** [review]
Reviewed changes and tested against 12.2.0.1 and 19.15

**Review by cofin — APPROVED** [review]

**Pull request merged**